app.py

```diff
@@ -1,12 +1,80 @@
-from flask import Flask, render_template, request, redirect, url_for
+import requests
+from flask_config import Config
+from flask import Flask, render_template, request, redirect, url_for, jsonify
 import session_items as session
 
 app = Flask(__name__)
-app.config.from_object('flask_config.Config')
+
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    return 'Hello World! <a href=http://127.0.0.1:5000/todo> todos</a>'
+
+
+@app.route('/todo', methods=['GET'])
+@app.route('/todo/', methods=['GET'])
+def todolist():
+    return render_template('index.html', todos=None)
```

Comment:
`todos=None` ? Did you forget to push some more changes?

You need to return a list of todos or you just get an error page.

Also don't forget the exercise brief asks you to define a class for the todo items.